pkg/asset/internal/templates.go

```diff
@@ -113,6 +113,8 @@ metadata:
   namespace: kube-system
   labels:
     k8s-app: kube-apiserver
+  annotations:
+    alpha.coreos.com/checkpoint: kube-apiserver
```

Comment:
This annotation needs to be moved under `spec.template.metadata` -- we want the annotation on the pod (and this is on the daemonset)